Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ui/ViewExtensions.kt

```diff
@@ -12,3 +13,6 @@ fun View.setGone(gone: Boolean) {
 fun View.setInvisible(invisible: Boolean) {
     visibility = if (invisible) View.INVISIBLE else View.VISIBLE
 }
+
+fun View.setOnClickListenerThrottled(interval: Long, listenerBlock: (View) -> Unit) =
```

Comment:
you can set an interval default value so you don't have to necessarily set it in the caller.

Reply:
Done 👍